release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,573 @@
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+* *TP*: _Technology Preview_
+* *GA*: _General Availability_
+* *-*: _Not Available_
+* *DEP*: _Deprecated_
+
+//TODO: remove anything that has been GA since 4.6
+
+.Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.7 |OCP 4.8 |OCP 4.9
+
+|Precision Time Protocol (PTP)
+|TP
+|TP
+|
+
+|`oc` CLI plug-ins
+|TP
+|GA
+|
+
+|Descheduler
+|GA
+|GA
+|
+
+|HPA for memory utilization
+|GA
+|GA
+|
+
+|Service Binding
+|TP
+|TP
+|
+
+|Raw Block with Cinder
+|TP
+|GA
+|
+
+|CSI volume snapshots
+|GA
+|GA
+|
+
+|CSI volume expansion
+|TP
+|TP
+|
+
+|vSphere Problem Detector Operator
+|GA
+|GA
+|
+
+|CSI Azure Disk Driver Operator
+|-
+|TP
+|
+
+|CSI GCP PD Driver Operator
+|TP
+|GA
+|
+
+|CSI OpenStack Cinder Driver Operator
+|TP
+|TP
+|
+
+|CSI AWS EBS Driver Operator
+|TP
+|TP
+|
+
+|CSI automatic migration
+|-
+|TP
+|
+
+|CSI inline ephemeral volumes
+|TP
+|TP
+|
+
+|CSI vSphere Driver Operator
+|-
+|TP
+|
+
+|Automatic device discovery and provisioning with Local Storage Operator
+|TP
+|TP
+|
+
+|OpenShift Pipelines
+|TP
+|GA
+|
+
+|OpenShift GitOps
+|TP
+|GA
+|
+
+|OpenShift sandboxed containers
+|-
+|TP
+|
+
+|Vertical Pod Autoscaler
+|TP
+|GA
+|
+
+|Cron jobs
+|TP
+|GA
+|
+
+|PodDisruptionBudget
+|TP
+|GA
+|
+
+|Adding kernel modules to nodes with kvc
+|TP
+|TP
+|
+
+|Egress router CNI plug-in
+|TP
+|GA
+|
+
+|Scheduler profiles
+|TP
+|TP
+|
+
+|Non-preempting priority classes
+|TP
+|TP
+|
+
+|Kubernetes NMState Operator
+|TP
+|TP
+|
+
+|Assisted Installer
+|TP
+|TP
+|
+
+|AWS Security Token Service (STS)
+|TP
+|GA
+|GA
+
+|Kdump
+|TP
+|TP
+|
+
+|OpenShift Serverless
+|-
+|GA
+|
+
+|Serverless functions
+|-
+|TP
+|
+
+|====
+
+[id="ocp-4-9-known-issues"]
+== Known issues
+
+// TODO: This known issue should carry forward to 4.8 and beyond! This needs some SME/QE review before being updated for 4.9.
+* In {product-title} 4.1, anonymous users could access discovery endpoints. Later releases revoked this access to reduce the possible attack surface for security exploits because some discovery endpoints are forwarded to aggregated API servers. However, unauthenticated access is preserved in upgraded clusters so that existing use cases are not broken.
++
+If you are a cluster administrator for a cluster that has been upgraded from {product-title} 4.1 to 4.8, you can either revoke or continue to allow unauthenticated access. It is recommended to revoke unauthenticated access unless there is a specific need for it. If you do continue to allow unauthenticated access, be aware of the increased risks.
```

Comment:
Should 4.8 be updated to 4.9 here?

Reply:
I think so, but it's been in there a while and I want to ask an SME about it (see note on 506)